memo_app.rb

```diff
@@ -53,48 +64,60 @@ def find_memo(id)
 end
 
 post '/create' do
-  add_memo(params)
-  redirect '/'
+  if params['title'].size.positive?
```

Comment:
Railsだと、`if params['title'].present?` みたいに書けるんですが、素のRubyだと「空でない」を表現するのがちょっと遠回りになりますね。

ただ、size.positive? はちょっと長いので、僕だったらこう書くかな〜。

```suggestion
  if !params['title'].empty?
```

もしRuboCopに怒られるなら `if params['title'].empty?` として異常系を前に持ってくるか、`unless params['title'].empty?` にするか、どちらかですかね〜。